packages/cashier/src/pages/account-transfer/account-transfer.tsx

```diff
@@ -132,7 +132,7 @@ export default connect(({ client, modules }: TRootStore) => ({
     is_loading: modules.cashier.general_store.is_loading,
     is_switching: client.is_switching,
     is_transfer_confirm: modules.cashier.account_transfer.is_transfer_confirm,
-    is_transfer_locked: modules.cashier.general_store.is_transfer_locked,
+    is_transfer_locked: modules.cashier.account_transfer.is_transfer_locked,
```

Comment:
use connect(({ modules: { cashier } }) so that you do not have to repeat modules.cashier in your code.

Reply:
@hirad-rewok Yes you are right, But I think that change would be out of this PR scope.
Also, we are going to remove the `connect` method in the `cashier` package after #6339 is merged.